lib/matplotlib/type1font.py

```diff
@@ -146,47 +145,52 @@ def _tokens(cls, text):
         A PostScript tokenizer. Yield (token, value) pairs such as
         (_TokenType.whitespace, '   ') or (_TokenType.name, '/Foobar').
         """
+        # Preload enum members for speed.
+        whitespace = _TokenType.whitespace
+        name = _TokenType.name
+        string = _TokenType.string
+        delimiter = _TokenType.delimiter
+        number = _TokenType.number
         pos = 0
         while pos < len(text):
-            match = (cls._comment_re.match(text[pos:]) or
-                     cls._whitespace_re.match(text[pos:]))
+            match = cls._whitespace_or_comment_re.match(text, pos)
             if match:
-                yield (_TokenType.whitespace, match.group())
-                pos += match.end()
+                yield (whitespace, match.group())
+                pos = match.end()
             elif text[pos] == b'(':
                 start = pos
                 pos += 1
                 depth = 1
                 while depth:
-                    match = cls._instring_re.search(text[pos:])
+                    match = cls._instring_re.search(text, pos)
```

Comment:
Seems like a lot of this is untested, so its hard to tell if the micro-optimization works....

Reply:
good catch, should be fixed now...